torch/_tensor.py

```diff
@@ -86,9 +86,14 @@ def __deepcopy__(self, memo):
                         self.requires_grad,
                         self._backward_hooks)
                 else:
-                    new_tensor = self.new()
+                    if self.is_conj():
+                        new_tensor = self.conj().new()
+                    else:
+                        new_tensor = self.new()
```

Comment:
right it's a waste. I only did it because `new()` can't handle the conjugate bit, and there's no way to directly set and unset the conj bit for tensors in python.